### ISSUE TITLE
Keyboard navigation in one trust privacy preference center

### ISSUE THREAD

**Issue by KirstyScott** [visitscotland/vs-component-library#125]
To recreate:

1 - Visit visitscotland.com/policies/privacy/cookies

2 - Click on 'Manage Cookie Settings' , the privacy center modal window opens. 

3 - Use keyboard tabbing to navigate the interactive elements. 

4 - Tab onto the list of tabs on the left hand side, including 'Your Privacy, Necessary cookies, Analytical cookies ... '

Given a user is trying to tab down this list. When they use the down arrow keys. Then the page behind the modal scrolls. The user needs to use the left and right arrow keys to navigate up and down a vertical collection, and this is not intuitive.

Expected behavior: Keyboard users should be able to use the up and down arrow keys to navigate the vertical tab collection.
